pkg/internal/token/execCredentialPlugin.go

```diff
@@ -52,8 +66,9 @@ func (p *execCredentialPlugin) Do(ctx context.Context) error {
 		klog.V(5).Infof("failed to retrieve cached record: %s", err)
 	}
 
-	cred, err := p.newCredentialFunc(record, p.o)
+	cred, err := p.newCredentialFunc(record, p.popTokenCachedRecord, p.o)
 	if err != nil {
+		klog.V(5).Infof("failed to create credential: %s", err)
```

Comment:
is this log necessary? wouldn't the returned error in next line be sufficient?

Reply:
removed